code-reviews/example2/math_operations.py

```diff
@@ -0,0 +1,16 @@
+# Common mathemmatical operations
+
+def add(a, b):
+    return a + b
+
+def subtract(a, b):
+    return a - b
+
+def multiply(a, b):
+    return a * b
+
+
+def divide(a, b):
+    if a == 0:
+        raise ValueError("Cannot divide by zero")
+    return a / b
```

Comment:
I think at the end you should call one of the functions maybe for illustration

Reply:
Agree, could also be a comment above the function to demonstrate usage